packages/table/components/Table.tsx

```diff
@@ -73,7 +82,22 @@ export class Table<T> extends React.PureComponent<ITableProps, {}> {
     );
   }
 
+  private onScrollbarPresenceChange({ horizontal }) {
```

Comment:
We're following up to add the shadow on scroll with [DCOS-40413](https://jira.mesosphere.com/browse/DCOS-40413). Is this code still necessary for this PR?

Reply:
In this case, we are only displaying the shadow when `onScrollbarPresenceChange` is `true` that way when we have few columns without scroll we don't show the shadow as per request from the UX team.

DCOS-40413 is still needed in the future.